> This should be the last resort. Is it really not possible to stay with 10?

```
/nix/store/kg2s3k77ifsyd59ac08mjmmjjmsvgsvg-gcc-10.2.0/include/c++/10.2.0/backward/backward_warning.h:32:2: warning: #warning This file includes at least one deprecated or antiquated header which may be removed without further notice at a future date. Please use a non-deprecated interface with equivalent functionality instead. For a listing of replacement headers and interfaces, consult the file backward_warning.h. To disable this warning use -Wno-deprecated. [-Wcpp]
   32 | #warning \
      |  ^~~~~~~
In file included from /nix/store/kg2s3k77ifsyd59ac08mjmmjjmsvgsvg-gcc-10.2.0/include/c++/10.2.0/ext/hash_map:64,
                 from ../subprojects/libqalculate/libqalculate/Calculator_p.h:30,
                 from ../subprojects/libqalculate/libqalculate/Calculator-definitions.cc:63:
/nix/store/kg2s3k77ifsyd59ac08mjmmjjmsvgsvg-gcc-10.2.0/include/c++/10.2.0/backward/hashtable.h: In instantiation of '__gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::size_type __gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::_M_bkt_num_key(const key_type&, std::size_t) const [with _Val = std::pair<Unit* const, bool>; _Key = Unit*; _HashFcn = __gnu_cxx::hash<Unit*>; _ExtractKey = std::_Select1st<std::pair<Unit* const, bool> >; _EqualKey = std::equal_to<Unit*>; _Alloc = std::allocator<bool>; __gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::size_type = long unsigned int; __gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::key_type = Unit*; std::size_t = long unsigned int]':
/nix/store/kg2s3k77ifsyd59ac08mjmmjjmsvgsvg-gcc-10.2.0/include/c++/10.2.0/backward/hashtable.h:596:30:   required from '__gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::size_type __gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::_M_bkt_num_key(const key_type&) const [with _Val = std::pair<Unit* const, bool>; _Key = Unit*; _HashFcn = __gnu_cxx::hash<Unit*>; _ExtractKey = std::_Select1st<std::pair<Unit* const, bool> >; _EqualKey = std::equal_to<Unit*>; _Alloc = std::allocator<bool>; __gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::size_type = long unsigned int; __gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::key_type = Unit*]'
/nix/store/kg2s3k77ifsyd59ac08mjmmjjmsvgsvg-gcc-10.2.0/include/c++/10.2.0/backward/hashtable.h:519:32:   required from '__gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::iterator __gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::find(const key_type&) [with _Val = std::pair<Unit* const, bool>; _Key = Unit*; _HashFcn = __gnu_cxx::hash<Unit*>; _ExtractKey = std::_Select1st<std::pair<Unit* const, bool> >; _EqualKey = std::equal_to<Unit*>; _Alloc = std::allocator<bool>; __gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::iterator = __gnu_cxx::hashtable<std::pair<Unit* const, bool>, Unit*, __gnu_cxx::hash<Unit*>, std::_Select1st<std::pair<Unit* const, bool> >, std::equal_to<Unit*>, std::allocator<bool> >::iterator; __gnu_cxx::hashtable<_Val, _Key, _HashFcn, _ExtractKey, _EqualKey, _Alloc>::key_type = Unit*]'
/nix/store/kg2s3k77ifsyd59ac08mjmmjjmsvgsvg-gcc-10.2.0/include/c++/10.2.0/ext/hash_map:213:26:   required from '__gnu_cxx::hash_map<_Key, _Tp, _HashFn, _EqualKey, _Alloc>::iterator __gnu_cxx::hash_map<_Key, _Tp, _HashFn, _EqualKey, _Alloc>::find(const key_type&) [with _Key = Unit*; _Tp = bool; _HashFn = __gnu_cxx::hash<Unit*>; _EqualKey = std::equal_to<Unit*>; _Alloc = std::allocator<bool>; __gnu_cxx::hash_map<_Key, _Tp, _HashFn, _EqualKey, _Alloc>::iterator = __gnu_cxx::hashtable<std::pair<Unit* const, bool>, Unit*, __gnu_cxx::hash<Unit*>, std::_Select1st<std::pair<Unit* const, bool> >, std::equal_to<Unit*>, std::allocator<bool> >::iterator; __gnu_cxx::hash_map<_Key, _Tp, _HashFn, _EqualKey, _Alloc>::key_type = Unit*]'
../subprojects/libqalculate/libqalculate/Calculator-definitions.cc:3432:23:   required from here
/nix/store/kg2s3k77ifsyd59ac08mjmmjjmsvgsvg-gcc-10.2.0/include/c++/10.2.0/backward/hashtable.h:604:23: error: no match for call to '(const hasher {aka const __gnu_cxx::hash<Unit*>}) (Unit* const&)'
  604 |       { return _M_hash(__key) % __n; }
      |                ~~~~~~~^~~~~~~
ninja: build stopped: subcommand failed.
builder for '/nix/store/bqmbrh4gbgvkdj5wr4x56pnyz6732gkm-nasc-0.7.5.drv' failed with exit code 1
error: build of '/nix/store/bqmbrh4gbgvkdj5wr4x56pnyz6732gkm-nasc-0.7.5.drv' failed
```

but I maybe have found a patch to fix this.

--- a/pkgs/development/python-modules/keep/default.nix
+++ b/pkgs/development/python-modules/keep/default.nix
@@ -24,6 +24,7 @@ buildPythonPackage rec {
   ];
 
   # no tests
+  doCheck = false;
   pythonImportsCheck = [ "keep" ];
 
   meta = with lib; {

--- a/pkgs/development/python-modules/scrapy-deltafetch/default.nix
+++ b/pkgs/development/python-modules/scrapy-deltafetch/default.nix
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchPypi, buildPythonPackage, pytest, scrapy, bsddb3 }:
+{ stdenv, lib, fetchPypi, buildPythonPackage, scrapy, bsddb3 }:
 
 buildPythonPackage rec {
   pname = "scrapy-deltafetch";
@@ -11,7 +11,9 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ bsddb3 scrapy ];
 
-  checkInputs = [ pytest ];
+  # no tests
+  doCheck = false;
+  pythonImportsCheck = [ "scrapy_deltafetch" ];
 
   meta = with lib; {
     description = "Scrapy spider middleware to ignore requests to pages containing items seen in previous crawls";

--- a/pkgs/development/python-modules/scrapy-splash/default.nix
+++ b/pkgs/development/python-modules/scrapy-splash/default.nix
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchPypi, buildPythonPackage, pytest, hypothesis, scrapy }:
+{ stdenv, lib, fetchPypi, buildPythonPackage, scrapy, six }:
 
 buildPythonPackage rec {
   pname = "scrapy-splash";
@@ -9,7 +9,11 @@ buildPythonPackage rec {
     sha256 = "1dg7csdza2hzqskd9b9gx0v3saqsch4f0fwdp0a3p0822aqqi488";
   };
 
-  checkInputs = [ pytest hypothesis scrapy ];
+  propagatedBuildInputs = [ scrapy six ];
+
+  # no tests
+  doCheck = false;
+  pythonImportsCheck = [ "scrapy_splash" ];
 
   meta = with lib; {
     description = "Scrapy+Splash for JavaScript integration";

